docs/modules/ROOT/pages/cli/configuration.adoc

```diff
@@ -0,0 +1,103 @@
+= CLI Configuration
+
+The kubectl-archive plugin supports configuration through environment variables and CLI flags, following the same pattern as kubectl and other plugins.
+
+== Configuration Precedence
+
+Configuration values are resolved in the following order (highest to lowest priority):
+
+. *Plugin CLI Flags* - Plugin-specific command line arguments
+. *Environment Variables* - System environment variables
+. *Default Values* - Built-in defaults
+
+== Global Configuration Parameters
+
+All configuration can be set using environment variables with the prefix `KUBECTL_PLUGIN_ARCHIVE_`:
+
+[cols="1,2,1,1"]
+|===
+|Environment Variable |Description |Default |CLI Flag
+
+|`KUBECTL_PLUGIN_ARCHIVE_HOST`
+|KubeArchive API host:port
+|`https://localhost:8081`
+|`--host`
+
+|`KUBECTL_PLUGIN_ARCHIVE_CERT_PATH`
+|Path to KubeArchive API certificate file
+|(none)
+|`--kubearchive-certificate-authority`
+
+|`KUBECTL_PLUGIN_ARCHIVE_TLS_INSECURE`
+|Skip TLS certificate verification for the KubeArchive API server
+|`false`
+|`--kubearchive-insecure-skip-tls-verify`
+
+|`KUBECTL_PLUGIN_ARCHIVE_TOKEN`
+|Bearer token for KubeArchive API authentication
+|(from kubeconfig)
+|(uses kubectl's `--token` flag)
+
+|===
+
+== Authentication
+
+The KubeArchive API only supports bearer token authentication. If your kubeconfig uses client certificates or other authentication methods, you must provide a bearer token from a service account in the cluster with sufficient permissions to perform the requested operations (`get`, `list`, `logs`).
```

Comment:
```suggestion
The KubeArchive API only supports bearer token authentication. Provide a bearer token from a service account in the cluster with sufficient permissions to perform the requested operations (`get`, `list`, `logs`).
```

Reply:
Ah, I see. Then its fine.

Reply:
I've reformatted because it had a really long line but I think that the message is still meaninful